lib/xgu/xgux.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __XGUX_H
+#define __XGUX_H
+
+#include "xgu.h"
+
+inline
+void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int count) {
+    uint32_t *p = pb_begin();
+    p = xgu_begin(p, mode);
+    p = xgu_draw_arrays(p, start, count);
+    p = xgu_end(p);
+    p = pb_end(p);
```

Comment:
This needs a rebase to newer pbkit code. As is, `pb_end` returns void, so it actually breaks compilation

Reply:
Fixed. :+1: